### PR TITLE
Update issue templates to clarify intended usage

### PR DESCRIPTION
**Pull request by eldering**
People are often not removing the boilerplate text and quoting their text, replicating the boilerplate quoted text.

### CODE REVIEW

**Comment by nickygerritsen** [comment]
I believe you can put the boiler plate in `<--` or something to make it a comment so you don't even have to remove it

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -7,25 +7,36 @@ assignees: ''
...
 
 ### The goal you want to achieve
-> Please elaborate on the (larger, higher level) goal you want to achieve with this enhancement, so we have a good understanding what this feature would be useful for and how it fits in DOMjudge as a whole.
+<!--
+Please elaborate on the (larger, higher level) goal you want to achieve with this enhancement, so we have a good understanding what this feature would be useful for and how it fits in DOMjudge as a whole.
+-->
 
 ### Expected behaviour
-> Replace this line with what you would expect to happen.
-> Please describe the workflow how you want this feature to work with expected start URL
```
Comment (vmcj) [inline .github/ISSUE_TEMPLATE/feature_request.md:27]: This sentence has been poorly formulated.

**Review by vmcj — APPROVED** [review]
Not worse, so fine by me.

**Review by meisterT — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (eldering) [reply .github/ISSUE_TEMPLATE/feature_request.md:27]: I've updated the wording.

**Review by Kevinjil — APPROVED** [review]

**Comment by Kevinjil** [comment]
nit: perhaps `Write x below` is more suited than `Replace this line with x` in a commented section description.

**Pull request merged**